code/__DEFINES/antagonists.dm

```diff
@@ -217,22 +217,28 @@ GLOBAL_LIST_INIT(ai_employers, list(
 /// Checks if the given mob is a traitor
 #define IS_TRAITOR(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/traitor))
 
+/**
+ * Cult checks
+ */
+
 /// Checks if the given mob is a blood cultist
-#define IS_CULTIST(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/cult))
+#define IS_CULTIST(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/cult) || HAS_TRAIT(mob, TRAIT_ACT_AS_CULTIST))
```

Comment:
My 2c says you could instead just have `TRAIT_CULTIST` or `TRAIT_HERETIC`, replace relevant `IS_CULTIST` / `IS_HERETIC` checks (like in sword code) with `HAS_TRAIT(TRAIT_CULTIST)` etc, and then make those antag datums give those traits